metadat/developer/Nintendo - Game Boy Advance.dat

```diff
@@ -12187,15 +12187,15 @@ game (
 )
 
 game (
-	name "Pocket Monsters - Fire Red (Japan)"
+	name "Pocket Monsters - FireRed (Japan)"

```

Comment:
On http://datomatic.no-intro.org , you can submit to the Forums, and the Discussion queue.